src/client.cpp

```diff
@@ -881,6 +883,7 @@ int Client<IoType, SwitchDataIntegrity, SwitchActivityInfo, SwitchCycleDuration,
                 std::string hostport = sockaddr_to_hostport(p_client_bind_addr);
                 struct sockaddr_store_t unix_addr;
                 socklen_t unix_addr_len;
+#ifndef __windows__// AF_UNIX with DGRAM isn't supported in __windows__
```

Comment:
I think that it is better to print error message if user selects DGRAM local socket on Windows.

Reply:
It's a bit problematic since we don't have a flag for AF_UNIX, and default behavior is DGRAM.
The first place we know user have selected local sockets is in `resolve_sockaddr` func. 

Do you prefer adding to this function a condition like: 
`host !=NULL && sock_type != SOCK_DGRAM`?

Reply:
You can return an error right here:

```c++
                if (p_client_bind_addr->ss_family == AF_UNIX && g_pApp->m_const_params.sock_type == SOCK_DGRAM) { // Need to bind localy
#ifdef __windows__
                    log_error("unsupported combination....");
                    return ERROR;
#else // __windows__
                    if (p_client_bind_addr->addr_un.sun_path[0] == 0) { // no specific addr client_info was provoided
                        ...
                    }
#endif // __windows__
                }
```